tests/unittests/Snmp/Snmp_test.py

```diff
@@ -39,7 +39,7 @@ def tearDown(self):
 
     def test_load_pynetsnmp_if_available_as_first_choice(self):
         # This import should be fine
-        from nav.Snmp.pynetsnmp import *
+        import nav.Snmp.pynetsnmp as _
 
```

Comment:
Is this really a fix for python3?

Reply:
Python3 says "import * only allowed at module level" without this